eng/testing/tests.props

```diff
@@ -18,13 +18,20 @@
     <_withoutCategories Condition="'$(ArchiveTests)' == 'true'">$(_withoutCategories);IgnoreForCI</_withoutCategories>
     <_withoutCategories Condition="'$(TestScope)' == '' or '$(TestScope)' == 'innerloop'">$(_withoutCategories);OuterLoop</_withoutCategories>
     <_withoutCategories Condition="!$(_withCategories.Contains('failing'))">$(_withoutCategories);failing</_withoutCategories>
+    <MonoEnvOptions Condition="'$(MonoEnvOptions)' == '' and '$(MonoEnableInterpreter)' == 'true'">--interpreter</MonoEnvOptions>
```

Comment:
Is there a extensibility point to append to this env options? Like if I want to debug tests?

Reply:
Yeah, that would be one.  We may also want to provide extra params to the runtime.  Enable logging, for example.

Reply:
Let's introduce a variable or use this one to add the mono run settings.